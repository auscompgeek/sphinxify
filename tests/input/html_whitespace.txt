  /**
   * Get the raw output ratio of the duty cycle signal.
   *
   * <p> 0 means always low, an output equal to
   * GetOutputScaleFactor() means always high.
   *
   * @return output ratio in raw units
   */
