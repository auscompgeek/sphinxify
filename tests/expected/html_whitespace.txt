Get the raw output ratio of the duty cycle signal.

0 means always low, an output equal to
GetOutputScaleFactor() means always high.

:returns: output ratio in raw units
